core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -213,10 +217,19 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
     if (leaderId.isEmpty)
       leaderId = Some(member.memberId)
     members.put(member.memberId, member)
+    member.supportedProtocols.foreach(p => supportedProtocols(p._1) += 1)
```

Comment:
Minor nit: can we please avoid `_1` and `_2`? They make the code hard to read. The usual approach is to pattern match, For example:

```scala
member.supportedProtocols.foreach { case (foo, _) => ...
```

Reply:
done

Thank you!